Detectors/TPC/simulation/test/testTPCDigitContainer.cxx

```diff
@@ -29,41 +31,43 @@ namespace TPC {
   BOOST_AUTO_TEST_CASE(DigitContainer_test1)
   {
     const SAMPAProcessing& sampa = SAMPAProcessing::instance();
-    auto *mDigitContainer = new DigitContainer();
-
-    /// \todo add test with FairLink stuff if possible
-    std::vector<int> hitID = {1, 250, 3, 62, 1000};
-    //std::vector<int> MCevent = {1, 250, 3, 62, 1000};
-    //std::vector<int> MCtrack = {22, 3, 4, 23, 523};
-    std::vector<int> CRU     = {1, 23, 36, 53, 214};
-    std::vector<int> Time    = {231, 2, 500, 230, 1};
-    std::vector<int> Row     = {12, 5, 6, 2, 6};
-    std::vector<int> Pad     = {1, 15, 14, 23, 5};
-    std::vector<int> nEle    = {60, 100, 250, 1023, 2};
+    static FairRootManager *mgr = FairRootManager::Instance();
+    std::unique_ptr<DigitContainer> mDigitContainer = std::make_unique<DigitContainer>();
```

Comment:
In `testeTPCCommonMode` you made a simple instance. Do you need the digit container here on the head? Or can it also be a simple instance?
Please check also the other tests.